src/sentry/utils/services.py

```diff
@@ -559,7 +559,7 @@ def callback_timing(
             "status": primary_status,
             "primary": "true",
         },
-        **metric_kwargs,
+        **metric_kwargs,  # type: ignore
```

Comment:
Is this because of the mypy issue with expanding dictionaries?

Reply:
i don't even know how to interpret the error message, type inference appears to be complete bogus (explicit annotations don't help). you'd have to see for yourself